### PR TITLE
Fixes and cleanup

### PR DESCRIPTION
**Pull request by chrisgorgo**
I run all functional metrics on a dataset, cleaned up and rearranged the code, fixed some problems with the documentation.

Two most noticeable changes involve Frame dispalcement calculation. I changed the radius to 50 (from 80) which is consistent with FSL scripts, Power paper and documentation. I also fixed the total number of frame displacemnts which should be len(timeseries)-1 not  len(timeseries)-1. It would be great if someone could review these changes.

### CODE REVIEW

**Comment by bcipolli** [comment]
@chrisfilo Thanks for your changes; wish I had seen these before I started!

May I ask, how did you download the necessary files to run these analyses?  I can get some of the functional files via the web documentation, but nothing is there about the anatomical files.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+nibabel
+nitime
```
Comment (bcipolli) [inline requirements.txt:2]: please add pandas here as well.

**Comment by chrisgorgo** [comment]
I believe you need to provide your own files for this toolbox. The whole point is to apply these metrics to your own data.

**Comment by chrisgorgo** [comment]
@ccraddock are you sure you don't want to at least partially merge it? You know that the current code base is broken (and I'm not talking about the brain radius parameter).

**Pull request merged**